ui/src/modules/trackers-preview/trackers-preview.js

```diff
@@ -0,0 +1,170 @@
+/**
+ * WhoTracks.Me
+ * https://whotracks.me/
+ *
+ * Copyright 2017-present Ghostery GmbH. All rights reserved.
+ *
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0
+ */
+
+import { html, define, dispatch } from 'hybrids';
+
+export default define({
+  tag: 'ui-trackers-preview',
+  confirmDisabled: false,
+  stats: undefined,
+  domain: '',
+  render: ({ domain, confirmDisabled, stats }) => html`
+    <ui-header domain=${domain}>
+      <button class="svg-button" onclick="${(host) => dispatch(host, 'close')}">
+        <ui-icon name="close"></ui-icon>
+      </button>
+    </ui-header>
+
+    <main>
+      <h1>Trackers Preview</h1>
+
+      ${stats &&
+      html.resolve(
```

Comment:
indentation problem? why pretties does not catch it?

Reply:
Yes, it did (if it would be a problem here or anywhere else, the CI test would break). What do you mean about the "problem"? Prettier splits the conditions in templates into separate lines. It is how it works.